### DISCUSSION TITLE
Prisma migrate has stopped working for me

### DISCUSSION THREAD

**Discussion by sarcastic-verma** [prisma/prisma#18973] [Q&A]
Hey, I have been using prisma for a while and have never faced this issue but Idk for this new project I am working on just doesn't work with prisma. I recently add prisma to this project and whenever I create a new migration I get the following error, even on a new db:- 
`The database schema is not empty. Read more about how to baseline an existing production database: https://pris.ly/d/migrate-baseline`

After getting this error, if I check my postgres local client it has the tables in it (initially it was empty). 

I have tried deleting the migrations folder and taking a new db but nothing works. Need help!

<img width="1614" alt="image" src="https://user-images.githubusercontent.com/50954641/234857686-9554f2f2-3bb5-47ad-8cd3-547b207eb764.png">
Above is a fresh command I ran for a new db "yessss"(sry for the name, am frustrated a bit here). Also the schema it created in the "yessss" db doesn't have a migrations table.

**Comment by ludralph** [comment]
Hi @sarcastic-verma 👋 

Did you follow the [steps](https://www.prisma.io/docs/guides/migrate/developing-with-prisma-migrate/add-prisma-migrate-to-a-project#overview-of-the-steps) to baseline an existing database from the documentation?

<img width="779" alt="image" src="https://user-images.githubusercontent.com/13855608/234859279-e772b357-bfb4-4ffc-ad3c-e84da4b20aaa.png">

**Reply by sarcastic-verma** [reply]
This worked, the reason I didn't try it before was that I am currently doing local dev only and expected it work on a new database.

Can you please shed some light onwhy was prisma migrate —init {name} cmd creating the tables(without migration table) and throwing error?

**Reply by sarcastic-verma** [reply]
Getting this error now, if I try to run the migration.
<img width="950" alt="image" src="https://user-images.githubusercontent.com/50954641/234863039-5f812866-85d5-4dd1-af6e-1cd2137a5445.png">

**Reply by ludralph** [reply]
Can you delete the previously created tables and the _prisma_migrations  and start afresh? The previous command before you started baselining created the tables?

**Reply by sarcastic-verma** [reply]
Just tried this, doing so created no tables now

**Reply by sarcastic-verma** [reply]
I tried applying this migration to a new table using `prisma migrate dev`, which again gave the same error

**Reply by ludralph** [reply]
Can you try [resetting](https://www.prisma.io/docs/concepts/components/prisma-migrate/migrate-development-production#reset-the-development-database) the database?

**Reply by mastoj** [reply]
Did you find a solution to this? I experience the exact same thing:

* Fresh vercel postgres
* Run `prisma migrate dev` first time
* Run `prisma migrate --create-only` fails after that.

**Reply by mastoj** [reply]
Here is a screen showing that my first migrate succeeds and the second fails for no apparent reason

![prisma](https://github.com/prisma/prisma/assets/501424/11008d3f-aa48-4a66-afd8-7d93c6c3d090)

**Reply by ludralph** [reply]
@mastoj, for vercel postgres, this [comment](https://github.com/orgs/vercel/discussions/2515#discussioncomment-5954253) specifies a workaround to help you overcome this issue.

**Reply by reza-ebrahimi** [reply]
any workaround on this? Its also not possible to reset the database. The error `The database schema is not empty` is blocked most of the commands.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
